terraform/data/bq_views/events/unique_events.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+  delivery_id,
+  signature,
+  received event,
```

Comment:
I think you are missing a comma after received. 

Looking at the group by received appears to be a separate column. When I first read this I thought you were renaming the column e.g. `received as event` but my guess is you just missed the comma.